home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/RoleEnum.java

```diff
@@ -9,7 +9,9 @@
 
     ADMIN("admin"),
     USER("user"),
```

Comment:
As Oleksandr said that we can delete role User

Reply:
It should be deleted in role matrix impl